sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/TransactionOverview.kt

```diff
@@ -6,7 +6,7 @@ package cash.z.ecc.android.sdk.model
  * Note that both sent and received transactions will have a positive net value.  Consumers of this class must
  */
 data class TransactionOverview internal constructor(
-    internal val id: Long,
+    val id: Long,
```

Comment:
I had to make this visible again due to ECC Wallet integration issues.  I filed a followup issue to improve how we create stable IDs for Transactions.  https://github.com/zcash/zcash-android-wallet-sdk/issues/754